src/ape_plugins/_cli.py

```diff
@@ -22,38 +22,89 @@ def cli():
     """
 
 
-@cli.command(name="list", short_help="List installed plugins")
+def _remove_prefix_add_version(set_convert, _prefix, version):
+    output = set()
+    text = None
+    if set_convert:
+        for i in set_convert:
+            if i.startswith(_prefix):
+                text = i[len(_prefix) :]
+            if version:
+                output.add(f"{text}    ({version})")
+            else:
+                output.add(text)
+        return output
+    else:
+        return None
+
+
+@cli.command(name="list", short_help="Display all plugins installed and available (including Core)")
 @click.option(
     "-a",
     "--all",
     "display_all",
     default=False,
     is_flag=True,
-    help="Display all plugins (including Core)",
+    help="Display all plugins installed and available (including Core)",
 )
 @ape_cli_context()
 def _list(cli_ctx, display_all):
-    plugins = []
+    installed_second = set()
+    installed_first = set()
+    installed_third = set()
+    version = None
+    _prefix = "ape_"
```

Comment:
```suggestion
    prefix = "ape_"
```

Any reason this is underscored?

Reply:
what is convention with _var? Do you have a resource for me to read?

I want to only use this variable within the _list method and _format_output method. 

It is a way for me/us to understand it is a private variable. From what I've read in Python naming convention

Reply:
If the method is defined at the module level or the class name, use the underscore prefix to indicate that the variable is protected.  If the variable is only defined locally within a the implementation of a method, you don't need to do that